eth2/utils/eth2_wallet/src/json_wallet/mod.rs

```diff
@@ -0,0 +1,60 @@
+use serde::{Deserialize, Serialize};
+use serde_repr::*;
+use std::convert::TryFrom;
+
+pub use eth2_keystore::json_keystore::{
+    Aes128Ctr, ChecksumModule, Cipher, CipherModule, Crypto, EmptyMap, EmptyString, Kdf, KdfModule,
+    Scrypt, Sha256Checksum,
+};
+pub use uuid::Uuid;
+
+#[derive(Debug, Clone, PartialEq, Serialize, Deserialize)]
+#[serde(deny_unknown_fields)]
+pub struct JsonWallet {
+    pub crypto: Crypto,
+    pub name: String,
+    pub nextaccount: u32,
```

Comment:
`nextaccount` is an optional field, but if it is missing for the json will error.

Reply:
It's not super clear if this is optional or not. The spec says:

> This element MAY be present if the wallet type requires it. If present it MUST be a non-negative integer.

I would argue that a HD wallet "requires" a `nextaccount` otherwise it's not really hierarchical. 

Also, according to [this comment](https://github.com/ethereum/EIPs/pull/2386#issuecomment-626130947) non-HD wallets are going to be removed so I can come back to this after the EIP gets finalized, if so happens `nextaccount` isn't required.

Reply:
I've added a todo with a link to this comment :)